docs/modules/matrix.md

```diff
@@ -0,0 +1,77 @@
+# matrix Module
```

Comment:
Would a better name be `matrixkdb` even? There are many types of matrices, I mean.